batch-index-files/decompress_json_files.py

```diff
@@ -0,0 +1,55 @@
+import gzip
```

Comment:
This is kind of yuck.  I was hoping to just use `bgzip -d <s3-path>` similar to what we do for compression, but bgzip doesn't seem to want to work that way.  For now, the code downloads the file in question, runs bgzip -d on the local file then uploads the decompressed version and then deletes the compressed version from s3.